components/suggest/src/suggestion.rs

```diff
@@ -109,6 +109,37 @@ impl Ord for Suggestion {
     }
 }
 
+impl Suggestion {
+    /// Get the URL for this suggestion, if present
+    pub fn url(&self) -> Option<&str> {
```

Comment:
```suggestion
    pub(crate) fn url(&self) -> Option<&str> {
```

Another loose convention I've been following—I'm curious what you think of it, actually, and if it's worth continuing? 😄—is using `pub(crate)` for crate-internal symbols, and `pub` for symbols that are exposed in the UDL.

I thought it might make it a little easier to see what's exposed to foreign callers, but I wonder if that's actually true. WDYT?

Reply:
I wonder about this a lot myself and have gone back and forth on what's best.  My current practice is to label everything `pub` if it's not private to the module, then let the higher-level modules control the if they want to do `use vs pub use`.  It's less typing and when I need to see what's exposed I go to `lib.rs` anyways.  I'm not confident that this is the Right Way, but it's worked okay for me.

Reply:
That sounds perfect, thanks! I like your approach a lot more 😁